### PR TITLE
Updated to PHP 7.3.19

### PR DESCRIPTION
**Pull request by chrisbarnett1**
I've updated the spec file to use the latest version of PHP.

Release notes:
https://www.php.net/ChangeLog-7.php#7.3.19

### CODE REVIEW

File: php73.spec [hunk 1]
```diff
@@ -62,7 +62,7 @@
 
 Summary: PHP scripting language for creating dynamic web sites
 Name: php73
-Version: 7.3.18
+Version: 7.3.19
 Release: 2%{?dist}
```
Comment (jeffsheltren) [inline php73.spec:66]: Please reset `Release` to `1%{?dist}`

**Comment by carlwgeorge** [comment]
php73-7.3.19-1.el7.ius has been published to ius-testing.

**Comment by ryandiamond23** [comment]
Any idea when these packages will be moved to the live IUS repo?

**Comment by chrisbarnett1** [comment]
Thanks for merging. I’ve tested the build from the testing repo on a few servers running WordPress and it is working well for me

**Comment by carlwgeorge** [comment]
@ryandiamond23 Packages typically stay in the testing repository for about a week.  This is mentioned in our [FAQs](https://ius.io/faq#how-long-do-updates-have-to-wait-in-the-testing-repository-before-being-promoted-to-the-main-repository).

@chrisbarnett1 Thanks for testing.  I've promoted php73-7.3.19-1.el7.ius to the main repository.  See #23 for the status on 7.3.20, which was released upstream today.

**Pull request merged**